bokeh/charts/builder/bar_builder.py

```diff
@@ -37,7 +37,40 @@
 
 def Bar(values, cat=None, stacked=False, xscale="categorical", yscale="linear",
         xgrid=False, ygrid=True, continuous_range=None, **kw):
+    """ Create a Bar chart using the `BarBuilder` to
```

Comment:
I don't think the backticks will do anything here. This
```
:class:`BarBuilder`
```
*might* generate a link, but I am not sure Sphinx will be smart enough in this case. If not, I would just leave the backticks off.